code/datums/quirks/positive_quirks.dm

```diff
@@ -207,6 +207,49 @@
 	// We've either added or removed TRAIT_NIGHT_VISION before calling this proc. Just refresh the eyes.
 	eyes.refresh()
 
+/datum/quirk/bilingual
+	name = "Bilingual"
+	desc = "Over the years you've picked up an extra language!"
+	icon = FA_ICON_GLOBE
+	value = 4
+	gain_text = span_notice("Some of the words of the people around you certainly aren't common. Good thing you studied for this.")
+	lose_text = span_notice("You seem to have forgotten your second language.")
+	medical_record_text = "Patient speaks multiple languages. Showoff."
+	var/list/possible_languages = list(
+		/datum/language/aphasia,
+		/datum/language/beachbum,
+		/datum/language/calcic,
+		/datum/language/draconic,
+		/datum/language/drone,
+		/datum/language/moffic,
+		/datum/language/monkey,
+		/datum/language/mushroom,
+		/datum/language/nekomimetic,
+		/datum/language/piratespeak,
+		/datum/language/shadowtongue,
+		/datum/language/slime,
+		/datum/language/sylvan,
+		/datum/language/terrum,
+		/datum/language/uncommon,
+		/datum/language/voltaic,
+		/datum/language/xenocommon,
```

Comment:
Xenocommon, uncommon, and drone should also be removed from this pool. 
- Xenocommon is another antag language + human tongues can't speak it
- Humans can't speak drone language either
- Uncommon is kinda Foreigner's thing

Reply:
Taking Foreigner removes common. I think it'd be nice to possibly give people both.

Reply:
I feel like that removes a lot of the fun of the quirk if people can just randomly ignore it and understand you (outside of the curator).

Reply:
I see.